coreos-installer

```diff
@@ -904,6 +904,11 @@ main() {
     # by a few of the functions below.
     DEST_DEV=$(cat /tmp/selected_dev)
     DEST_DEV=/dev/$DEST_DEV
+    if [ -L "${DEST_DEV}" ]; then
+	REAL_DEV=$(realpath $DEST_DEV)
+	echo "$DEST_DEV --> $REAL_DEV"
+	DEST_DEV=$REAL_DEV
```

Comment:
Minor: looks like tabs here instead of spaces? Madness!

Reply:
let me check, sounds crazy )

Reply:
emacs on test-machine had no config. fixed